ilc/common/environment.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
```

Comment:
can we use abstraction to read items from environment to not to put everything in object